src/mpi/romio/adio/common/ad_read.c

```diff
@@ -37,6 +37,12 @@ void ADIOI_GEN_ReadContig(ADIO_File fd, void *buf, int count,
 #ifdef AGGREGATION_PROFILE
     MPE_Log_event(5034, 0, NULL);
 #endif
+
+    if (count == 0) {
```

Comment:
@roblatham00 should I move this above the `MPL_Log_event`?

Reply:
we should get those MPE calls out of ROMIO
in the mean time, no, I think this logging call is observing the fact that something calls ADIO_GEN_ReadContig. zero check is ok where it is.